modules/cnf-topology-aware-lifecycle-manager-about-subscription-crs.adoc

```diff
@@ -6,16 +6,11 @@
 [id="talo-about-subscription-crs_{context}"]
 = Configuring Operator subscriptions for managed clusters that you install with {cgu-operator}
 
-{cgu-operator} can only approve the install plan for an Operator if the `Subscription` CR of the Operator contains a valid `status` field. You can use the following fields:
-
-* `status.state.AtLatestKnown` for the latest Operator version
-* `status.installedCSV` for a specific Operator version
+{cgu-operator} can only approve the install plan for an Operator if the `Subscription` CR of the Operator contains the `status.state.AtLatestKnown` field.
```

Comment:
Since this is the first occurrence of TALM in this module, consider expanding it; same for CR
```suggestion
{cgu-operator-first} can only approve the install plan for an Operator if the `Subscription` custom resource (CR) of the Operator contains the `status.state.AtLatestKnown` field.
```